src/ui/component/commentCreate/view.jsx

```diff
@@ -0,0 +1,84 @@
+// @flow
+import React from 'react';
+import { FormField, Form } from 'component/common/form';
+import Button from 'component/button';
+import ChannelSection from 'component/selectChannel';
+import usePersistedState from 'util/use-persisted-state';
+
+// props:
+type Props = {
+  uri: string,
+  claim: StreamClaim,
+  createComment: (string, string, string) => void,
+};
+
+export function CommentCreate(props: Props) {
+  const COMMENT_ACKNOWLEDGED = 'COMMENT_ACKNOWLEDGED';
+  const { createComment, claim } = props;
+  const { claim_id: claimId } = claim;
+  const [commentValue, setCommentValue] = usePersistedState(`comment-${claimId}`, '');
+  const [commentAck, setCommentAck] = usePersistedState(COMMENT_ACKNOWLEDGED, false);
+  const [channel, setChannel] = usePersistedState('COMMENT_CHANNEL', 'anonymous');
+
+  function handleCommentChange(event) {
+    setCommentValue(event.target.value);
+  }
+
+  function handleChannelChange(channel) {
+    setChannel(channel);
+  }
+
+  function handleCommentAck(event) {
+    setCommentAck(true);
+  }
+  function handleSubmit() {
+    if (channel !== 'new' && commentValue.length) createComment(commentValue, claimId, channel);
```

Comment:
The `new` here is a value that comes from `<ChannelSection>`, which _is_ the right time to use a `const` (otherwise, when editing `ChannelSection`, I'd have to check every place it is embedded and manually read all of that code to see if `"new"` is being used or checked against).